pkg/server/routes.go

```diff
@@ -46,18 +46,27 @@ func (s *Server) routes(
 	}
 
 	// set up CEDAR client
-	cedarEasiClient := cedareasi.NewTranslatedClient(
+	var cedarEasiClient cedareasi.Client
+	connectedCedarEasiClient := cedareasi.NewTranslatedClient(
 		s.Config.GetString("CEDAR_API_URL"),
 		s.Config.GetString("CEDAR_API_KEY"),
 	)
+	if s.environment.Deployed() {
+		s.CheckCEDAREasiClientConnection(connectedCedarEasiClient)
+	}
+	if s.environment.Local() {
```

Comment:
I think we'll probably want to be able to toggle this on and off with an environment variable. `CEDAR_CLIENT=local` or something? I imagine there will be times when we want to be able to actually talk to CEDAR when developing that functionality.

Reply:
It seems equally easy to just modify this line here when we're working on CEDAR stuff, so I'm inclined to leave this alone for now